HW04/Kate Fedotova/array_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rspec'
+require 'rubocop-rspec'
+require_relative 'arr.rb'
+
+# rubocop:disable Metrics/BlockLength
+describe Array do
+  describe 'my_each' do
```

Comment:
We use this style to describe method name http://www.betterspecs.org/#describe
```suggestion
  describe '#my_each' do
```